compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/SymbolFactory.java

```diff
@@ -233,6 +235,18 @@ public Symbol getBCompiledSymbol(BSymbol symbol, String name) {
             return createXMLNamespaceSymbol((BXMLNSSymbol) symbol);
         }
 
+        if (symbol.kind == SymbolKind.RESOURCE_PATH_IDENTIFIER_SEGMENT) {
```

Comment:
Can't change now, but shouldn't this have been `RESOURCE_PATH_NAME_SEGMENT`?

Reply:
Yes. I kept it as it is